backend/src/ml_space_lambda/authorizer/lambda_function.py

```diff
@@ -283,8 +285,15 @@ def lambda_handler(event, context):
             ) and Permission.ADMIN in user.permissions:
                 policy_statement["Effect"] = "Allow"
             elif requested_resource == "/project" and request_method == "POST":
-                # Anyone can create a project
-                policy_statement["Effect"] = "Allow"
+                # Get the latest app config
+                app_config = AppConfigurationModel.from_dict(
+                    app_configuration_dao.get(configScope="global", num_versions=1)[0]
+                )
+                # Check if project creation is admin only; if not, anyone can create a project
+                if (
+                    app_config.configuration.project_creation.admin_only and Permission.ADMIN in user.permissions
```

Comment:
We can do the admin check before fetching the config and save a DDB call.

Reply:
Good idea, will do